test/dotnet-publish.Tests/PublishPortableTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Publish.Tests
+{
+    public class PublishPortableTests : TestBase
+    {
+        private static readonly IEnumerable<Tuple<string, string>> ExpectedRuntimeOutputs = new[] {
+            Tuple.Create("debian-x64", "libuv.so"),
+            Tuple.Create("rhel-x64", "libuv.so"),
+            Tuple.Create("osx", "libuv.dylib"),
+            Tuple.Create("win7-arm", "libuv.dll"),
+            Tuple.Create("win7-x86", "libuv.dll"),
+            Tuple.Create("win7-x64", "libuv.dll")
+        };
+
+        [Fact]
+        public void PortableAppWithRuntimeTargetsIsPublishedCorrectly()
```

Comment:
I'd like to have build portable test, which checks that libuv is copied for current runtime.